tests/raftstore/cluster.rs

```diff
@@ -163,10 +167,11 @@ impl<T: Simulator> Cluster<T> {
     }
 
     pub fn call_command(&self,
+                        node_id: u64,
```

Comment:
The RaftCmdRequest header's peer contains store_id, seem no need add this argument.